tests/core/asyncio/test_basics.py

```diff
@@ -9,14 +9,17 @@
     DummyResponse,
 )
 from lahja import (
+    AsyncioEndpoint,
     BaseEvent,
-    Endpoint,
     UnexpectedResponse,
 )
 
 
 @pytest.mark.asyncio
-async def test_request(endpoint):
+async def test_request(endpoint, event_loop):
+    import logging
+    logging.info('LOOP IN TEST: %s', id(event_loop))
```

Comment:
Was this meant to stay here? If yes, why not import `logging` at the top?

Reply:
no, should be removed.